src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -9,34 +9,33 @@ import {
   isInterleaved,
   SIGNAL_ABORTED
 } from '../../loaders/utils';
+import { ColorPaletteExtension } from '../../extensions';
 
 const defaultProps = {
   pickable: { type: 'boolean', value: true, compare: true },
   onHover: { type: 'function', value: null, compare: false },
   contrastLimits: { type: 'array', value: [], compare: true },
-  colors: { type: 'array', value: [], compare: true },
   channelsVisible: { type: 'array', value: [], compare: true },
-  opacity: { type: 'number', value: 1, compare: true },
-  colormap: { type: 'string', value: '', compare: true },
```

Comment:
Don't these props still exist though?

Reply:
Yes but they are no longer on this layer actually by default as they are on the extensions.